docker/bookkeeper/entrypoint.sh

```diff
@@ -20,6 +20,7 @@ PRAVEGA_PATH=${PRAVEGA_PATH:-"pravega"}
 PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
 BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_AUTORECOVERY=${BK_AUTORECOVERY:-"false"}
+BK_useHostNameAsBookieID=${BK_useHostNameAsBookieID:-"false"}
```

Comment:
This is set to false by default where as the PR says. Please update the PR/the default value here:
> Add useHostNameAsBookieID=true to bk_server.conf to replace hostname with IP address so that bookie pods can process failover properly.

Reply:
Should I run the `system test` on my own? Or it is triggered by some label?